src/test/java/post/PostTest.java

```diff
@@ -5,7 +5,6 @@
 import org.testng.Assert;
 import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
-import routes.Get;
 import routes.Post;
 
 public class PostTest extends BaseTest {
```

Comment:
How do you create a user here?
It looks like you need to set addition params - http://dummy.restapiexample.com/create